src/App.tsx

```diff
@@ -186,11 +188,20 @@ export default function App() {
 			<table>
 				<tr>
 					<td><span className={"timeline-cell label"}>Frame</span></td>
-					<td><input className={"timeline-cell"} type={"text"} value={"0"} /></td>
+					<td><input className={"timeline-cell"} type={"number"} value={frameStart} min={"0"} onChange={(e) => 
+							{const inputInt = parseInt(e.target.value);
```

Comment:
this formatting looks rather ugly, I would put the curly braces on the outside lines

Reply:
resolved